Documentation/contributing/development/dev_setup.rst

```diff
@@ -147,13 +147,20 @@ directly and manually install Cilium:
         $ vagrant init cilium/ubuntu
         $ vagrant up
         $ vagrant ssh [...]
+        $ go get github.com/cilium/cilium
```

Comment:
This step should not be necessary, wasn't cilium source code available inside the VM?

Could you try running these steps in to directory where the source code lives? For example:

```
cd $GOPATH/src/github.com/cilium/cilium
vagrant init cilium/ubuntu
vagrant up
vagrant ssh [...]
...
```

Reply:
one issue here:
```
$ ~go/src/github.com/cilium/cilium# vagrant init cilium/ubuntu
==> vagrant: A new version of Vagrant is available: 2.2.9 (installed version: 2.2.7)!
==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

`Vagrantfile` already exists in this directory. Remove it before
running `vagrant init`.```

Reply:
Hmm that's a good point. I see that we have instructions how to use the Vagrantfile available in the project so I think we can have specific instructions for this one in here. Thanks for trying it out!